Modules/Core/Common/include/itkImageSink.hxx

```diff
@@ -94,12 +94,20 @@ ImageSink<TInputImage>
 template <typename TInputImage>
 void
 ImageSink<TInputImage>
-::Update( )
+::UpdateLargestPossibleRegion()
```

Comment:
Is this left empty on purpose?

Reply:
Good catch, missed in a rebase -- it should be calling `this->Update()`. I will push a follow-up.

Reply:
@thewtex This does not correspond to you comment.

There is still the default ProcessObject's Primary output, since we are not setting one in the filters now. The construction of it looks very odd:
https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Common/src/itkProcessObject.cxx#L67-L69